src/utils/date.ts

```diff
@@ -331,3 +331,26 @@ function getShortHumanizer(
 
   return humanizer(ms, opts);
 }
+
+export function formatTripDuration(
```

Comment:
Not sure where this function should be placed. Used for the details page (header and below map) and for the trip pattern in the search results view.

Reply:
I'm not sure either, but an alternative is to create a new file in the root utils folder 🤔 This might be fine though